src/components/CheckboxWithTooltip/CheckboxWithTooltipForMobileWebAndNative.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {View} from 'react-native';
+import Checkbox from '../Checkbox';
+import {propTypes, defaultProps} from './CheckboxWithTooltipPropTypes';
+import Growl from '../../libs/Growl';
+import CONST from '../../CONST';
+import compose from '../../libs/compose';
+import withWindowDimensions from '../withWindowDimensions';
+
+class CheckboxWithTooltipForMobileWebAndNative extends React.Component {
+    constructor(props) {
+        super(props);
+        this.onPress = this.onPress.bind(this);
+    }
+
+    componentDidUpdate() {
+        if (this.props.toggleTooltip) {
+            Growl.show(this.props.text, CONST.GROWL.WARNING, 3000);
+        }
+    }
+
+    /**
+     * Show warning modal on mobile devices since tooltips are not supported when checkbox is disabled.
+     */
+    onPress() {
+        if (this.props.toggleTooltip) {
+            Growl.show(this.props.text, CONST.GROWL.WARNING, 3000);
```

Comment:
I think in the context of this PR, the WARNING growl makes sense, but since this is a generic component, I don't know that it is good to have this default to WARNING. It would mean that any checkbox with a tooltip would always display the growl as a warning. Is that what we want?